docs/content/docs/developer-guide/contribute-to-flink-cdc.md

```diff
@@ -23,3 +23,62 @@ KIND, either express or implied.  See the License for the
 specific language governing permissions and limitations
 under the License.
 -->
+
+# Contributing
+
+Flink CDC is developed by an open and friendly community and welcomes anyone who wants to help out in any way.
+There are several ways to interact with the community and contribute to Flink CDC including asking questions, filing 
+bug reports, proposing new features, joining discussions on the mailing lists, contributing code or documentation, 
+improving website, testing release candidates and writing corresponding blog etc.
+
+## What do you want to do
+
+Contributing to Flink CDC goes beyond writing code for the project. Here are different opportunities to help the 
+project as follows.
+
+| Area            | Further information                                                                                                                                                                                                                                                                           |
+|:----------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| Report Bug      | To report a problem, open an issue in [Flink jira](https://issues.apache.org/jira/projects/FLINK/issues) and select `Flink CDC` in `Component/s`. Please give detailed information about the problem you encountered and, if possible, add a description that helps to reproduce the problem. |
+| Contribute Code | Read the <a href="#code-contribution-guide">Code Contribution Guide</a>                                                                                                                                                                                                                       |
+| Code Reviews    | Read the <a href="#code-review-guide">Code Review Guide</a>                                                                                                                                                                                                                                   |
+| Support Users   | Reply to questions on the flink user mailing list, check the latest issues in [Flink jira](https://issues.apache.org/jira/projects/FLINK/issues) for tickets which are actually user questions.                                                                                               |
```

Comment:
It's better to add a link to [mailing list](https://flink.apache.org/what-is-flink/community/#mailing-lists).